encrypted-media-respec.html

```diff
@@ -1498,17 +1498,19 @@
             The time represented by the <a def-id="expiration"></a> attribute MUST be earlier than the current time.
             All other keys in the session MUST have this status.
           </dd>
+          <dt>output-restricted</dt>
+          <dd>
+            There are output restrictions associated with the key which cannot currently be met.
```

Comment:
The descriptions for the two output-* values should be as close as possible. I suggest copying these (potentially slightly modified) to output-downscaled below. Probably with line 1510 in between them.